### DISCUSSION TITLE
Thank you Ronan!

### DISCUSSION THREAD

**Discussion by tvr256** [Rodemfr/MicronetToNMEA#70] [General]
I recently joined the MTN project when my Micronet NMEA sender stopped outputting data. Since joining, I've been absolutely amazed by the ingenuity and hard work that's been put into the project by everyone, especially Ronan.

I've raised a few minor bugs, and it's incredible how quickly Ronan has fixed them. When raising bugs its easy to sound like I'm complaining, but I really do appreciate the efforts that everyone has put into the project to make it a success. Thank you!

**Comment by Rodemfr** [comment]
Thanks a lot ! Reverse engineering Micronet is real fun and this gives me a bit the feeling to navigate when my boat is at the shipyard...
None of your bug reports have been taken as complains :-) More seriously, all the testing and reports by you, dwarning and j-lang and others are really helping to make MTN a usable project, this is gold. That's the magic of these open projects to bring people from all over the world on a common project.
